db.go

```diff
@@ -260,3 +275,38 @@ func (d *Db) insertHtlcResolution(ctx context.Context, htlc *HtlcInfo) error {
 
 	return err
 }
+
+func (d *Db) limitHTLCRecords(ctx context.Context) error {
+	query := `SELECT COUNT(id) from forwarding_history`
```

Comment:
This is a minor thing, but you could do this in a transaction. If someone would (manually) modify the db in between counting and deleting, too much may get deleted.

Although may be transaction still isn't good enough, because the table isn't fully locked.

I am fine with leaving it as is, just wanted to mention.

Reply:
Will leave a comment on the method! 

(personal opinion is that DB modifiers get what they deserve, _but_ definitely worth making a note of!)